tests/storage/hotplug.go

```diff
@@ -907,6 +909,100 @@ var _ = Describe(SIG("Hotplug", func() {
 			verifyNoVolumeAttached(vmi, "testvolume")
 		}
 
+		Context("Ephemeral Metrics", decorators.SigMonitoring, func() {
+
+			var (
+				vm *v1.VirtualMachine
+				sc string
+			)
+
+			BeforeEach(func() {
+				exists := false
+				sc, exists = libstorage.GetRWOFileSystemStorageClass()
+				if !exists {
+					Skip("Fail no filesystem storage class available")
+				}
+
+				vmi := libvmifact.NewCirros()
+				vm, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vmi)).Create(context.Background(), libvmi.NewVirtualMachine(vmi, libvmi.WithRunStrategy(v1.RunStrategyAlways)), metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				Eventually(matcher.ThisVM(vm)).WithTimeout(300 * time.Second).WithPolling(time.Second).Should(matcher.BeReady())
+			})
+
+			AfterEach(func() {
+				kvconfig.DisableFeatureGate(featuregate.DeclarativeHotplugVolumesGate)
+				kvconfig.DisableFeatureGate(featuregate.HotplugVolumesGate)
```

Comment:
it's `kvconfig.DisableFeatureGate(featuregate.HotplugVolumesGate)` in the case "should ignore delcarative hotplugs", maybe the reverted setting should be
`kvconfig.EnableFeatureGate(featuregate.HotplugVolumesGate)`